examples/plugin_new_macro.py

```diff
@@ -17,7 +20,8 @@
 # Define a new macro function
 def get_macros_with_date(self):
     macros = GET_MACROS_OLD(self)
-    macros['date'] = time.strftime('%m/%d/%Y')
+    # date is just a value so we can just put it in a lambda
+    macros['date'] = lambda: time.strftime('%m/%d/%Y')
```

Comment:
If it's just a value do we need the lambda? You said values don't have to be wrapped. Let's only introduce the laziness where we need it/it's useful.

Reply:
You are right here. I had made this change before I figured out that we can support callables as well as values.